data/weapons.txt

```diff
@@ -796,20 +796,20 @@ outfit "Torpedo Launcher"
 		"hit effect" "large explosion"
 		"die effect" "tiny explosion"
 		"inaccuracy" 5
-		"velocity" 7
+		"velocity" 3.5
 		"lifetime" 600
 		"reload" 120
 		"firing energy" 2
 		"firing heat" 45
-		"acceleration" .7
+		"acceleration" .6
```

Comment:
I'm wondering if just the turn rate change would be enough here, without the velocity change. Torpedos are already pretty easy for any starting ship but the Star Barge to avoid.

Reply:
to back that up:
nerfing turn rates makes them less effective against fast/agile ships that can dodge them; intended
nerfing velocity makes outrunning them easier, and gives AM more time to shoot them down, which is more a nerf to the torpedoes in general

if a faster ship could dodge a torpedo but not outrun it it'd be perfectly fine to me